enclave/src/test_proxy.rs

```diff
@@ -0,0 +1,128 @@
+use crate::polkadex;
+use crate::polkadex::PolkadexAccountsStorage;
+use codec::Encode;
+use sgx_rand::{Rng, SeedableRng, StdRng};
+use sgx_tstd::collections::HashMap;
+use sgx_types::{sgx_status_t, SgxResult};
+use sp_core::blake2_256;
+use sp_std::prelude::*;
+use std::{
+    sync::atomic::{AtomicPtr, Ordering},
+    sync::{Arc, SgxMutex},
+};
+static GLOBAL_ACCOUNTS_STORAGE: AtomicPtr<()> = AtomicPtr::new(0 as *mut ());
```

Comment:
Why is it necessary to redefine the atomic pointer here and not just take the actual storage from the polkadex file?
Doing it this way you are not testing the load and initialise function in the polkadex file. And these two functions should be tested as well.